docs/_guides/mango-queries.md

```diff
@@ -284,14 +284,31 @@ The Mango query language is generally very permissive, and allows you to write q
 
 As a straightforward example, if you query using the `_id` field, then the query planner will automatically map that directly to an `allDocs()` query. However, if you query for a field that isn't yet indexed, then it will simply use `allDocs()` to read in all documents from the database (!) and then filter in-memory. This can lead to poor performance, especially if your database is large.
 
-If you're ever wondering how the query planner is interpreting your query, you can enable debugging like so:
+If you're ever wondering how the query planner is interpreting your query, you can use the explain endpoint:
+
+```js
+db.find({
```

Comment:
Should this say `.find` ? How does this differ from the normal? Or should this say `.explain` ?